btrfs.go

```diff
@@ -84,6 +86,28 @@ func (f *FS) Info() (out Info, err error) {
 	return
 }
 
+type DevInfo struct {
+	UUID       UUID
+	BytesUsed  uint64
+	TotalBytes uint64
+	Path       string
+}
+
+func (f *FS) GetDevInfo(id uint64) (out DevInfo, err error) {
+	var arg btrfs_ioctl_dev_info_args
+	arg.devid = id
+
+	if err = ioctl.Do(f.f, _BTRFS_IOC_DEV_INFO, &arg); err != nil {
+		return
+	}
+	out.UUID = arg.uuid
+	out.BytesUsed = arg.bytes_used
+	out.TotalBytes = arg.total_bytes
+	out.Path = string(bytes.Trim(arg.path[:], "\x00"))
```

Comment:
I'm not sure if there a guarantee that the string will be fully padded by zeros at the end. There may be garbage after the first one.

I'd suggest something like this:

```go
path := arg.path[:]
if i := bytes.IndexByte(path, 0); i >= 0 {
   path = path[:i]
}
out.Path = string(path)
```

This deserves a helper, actually :sweat_smile: